<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1149</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lblOutput">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>200</y>
      <width>381</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnOk">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>140</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Ok</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>40</y>
      <width>211</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter a message</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtInput">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>40</y>
      <width>391</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QRadioButton" name="radio1">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>60</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>male</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radio2">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>110</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>female</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>210</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QRadioButton" name="radio3">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>160</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>n/a</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1149</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
